readme.md

```diff
@@ -1,11 +1,11 @@
 English ·
-[中文](./Translation:Chinese.md) ·
-[Español](./Translation:Spanish.md) ·
-[Português](./Translation:Portuguese.md) ·
-[Français](./Translation:French.md) ·
-[Русский](./Translation:Russian.md) ·
-[Deutsch](./Translation:German.md) ·
-[日本語](./Translation:Japanese.md)
+[中文](./Translation-Chinese.md) ·
```

Comment:
Links in Translation-*.md files need to be updated as well.

Reply:
Thank you @i-ky , I addressed them in the latest commit.